src/pymor/models/iosys.py

```diff
@@ -653,11 +656,8 @@ def _compute(self, solution=False, output=False, solution_d_mu=False, output_d_m
 
         # output computation
         if output:
-            Cx = self.C.apply(X, mu=mu).to_numpy()
-            if input is None or isinstance(self.D, ZeroOperator):
-                data['output'] = Cx
-            else:
-                raise NotImplementedError
+            D = LinearInputOperator(self.D)
+            data['output'] = self.C.apply(X, mu=mu).to_numpy() + D.apply(D.source.ones(1), mu=mu).to_numpy()
```

Comment:
I'm not sure if this does the right thing. Since `mu` contains a single time value (`t=0` in this case, I think), then `mu['input']` will also only be a single value.

I think using `time_stepper.iterate` is necessary. Also, it would allow not storing the whole state trajectory when `solution=False` and `output=True`.

Reply:
Oh yeah. Just working on that. What is the expected return format of `_compute`? A dict containing `VectorArrays` or Numpy arrays? I would think `VectorArrays` are expected but it was arrays before...

Reply:
The format of `data['solution']` and `data['output']` should be the same as, respectively, the return format of `solve` and `output`. At some point, we decided that inputs and outputs should be NumPy arrays.